service/grpc/topic.go

```diff
@@ -62,13 +65,38 @@ func (s *Server) OnTopicEvent(ctx context.Context, in *pb.TopicEventRequest) (*p
 	}
 	key := fmt.Sprintf("%s-%s", in.PubsubName, in.Topic)
 	if h, ok := s.topicSubscriptions[key]; ok {
+		data := interface{}(in.Data)
+		if len(in.Data) > 0 {
+			mediaType, _, err := mime.ParseMediaType(in.DataContentType)
+			if err == nil {
+				var v interface{}
+				switch mediaType {
+				case "application/json":
```

Comment:
Better to use constant here.

Reply:
@daixiang0 I've noticed you frequently nitpick this in reviews. My general rule of thumb is that I do not create a constant for literals that are only used once and that have meaning built in (such as a media type). Numbers tend to get constants just for added meaning/clarity. In other cases, it can make the code harder to grok. So I think the Dapr project should have clear guidelines on this.

Reply:
I agree with Phil here. I don't think we should add constants for literals. For numbers, it makes more sense. I really don't like when someone creates a constant with the same name as the value it holds, for example.

Reply:
At a previous employer, my team used Uber's Go style guide. We might be able to review this and "borrow" what we like. https://github.com/uber-go/guide/blob/master/style.md

Reply:
Agree.